azure/pipelines/dotnet/ci.yml

```diff
@@ -87,6 +91,16 @@ steps:
       externalFeedCredentials: ${{ parameters.ExternalFeedCredentials }}
       restoreArguments: --locked-mode
 
+- ${{ if and( eq(parameters.NoTests, false), ne(parameters.WithPlatinaTestInit, false) ) }}:
+  - task: PowerShell@2
+    displayName: 'Platina Begin Test Initialization'
+    inputs:
+      targetType: 'inline'
+      script: |
+        &'C:\Program Files\dotnet\dotnet.exe' msbuild /target:BeginPlatinaTestInit '${{ parameters.Solution }}'
+        # Ignore msbuild errors because targets might not be present in projects.
+        exit 0;
```

Comment:
Wouldn't it be better to allow errors? If you don't have the targets, it's probably because you are not using it, so then you should set the WithPlatinaInit flag to false?